### PR TITLE
Release hmis stable

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: docker/entrypoint.sh [hunk 1]
```diff
@@ -9,5 +9,17 @@ bundle config --global set build.sassc --disable-march-tune-native
...
 
+if [ ! -e /bundle/ruby/2.7.0/gems/seven_zip_ruby-1.3.0/lib/seven_zip_ruby/7z.so ]
+then
+  echo Reinstalling seven_zip_ruby because the dynamic link library is not in the correct place
+
+  bundle exec gem uninstall seven_zip_ruby
+
+  bundle install
+
+  cp /usr/local/lib/ruby/site_ruby/2.7.0/x86_64-linux-musl/seven_zip_ruby/7z.so \
```
Comment (eanders) [inline docker/entrypoint.sh]: @yakloinsteak won't this break those of us on arm?

File: docker/shell.entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+if [ ! -e /bundle/ruby/2.7.0/gems/seven_zip_ruby-1.3.0/lib/seven_zip_ruby/7z.so ]
+then
+  echo Reinstalling seven_zip_ruby because the dynamic link library is not in the correct place
+
+  bundle exec gem uninstall seven_zip_ruby
+
+  bundle install
+
+  cp /usr/local/lib/ruby/site_ruby/2.7.0/x86_64-linux-musl/seven_zip_ruby/7z.so \
```
Comment (eanders) [inline docker/shell.entrypoint.sh]: same here?

File: docker/entrypoint.sh (hunk 1 above)
Comment (yakloinsteak) [reply docker/entrypoint.sh]: I'll add another check, but I'm not sure.

Comment (yakloinsteak) [reply docker/entrypoint.sh]: Where does the 7z.so file end up on an M1? I need to know that to write that check correctly.

Comment (yakloinsteak) [reply docker/entrypoint.sh]: pushed a small improvement.  Could also not autoload that library in development.

**Pull request merged**